### PR TITLE
Added header section

### PR DESCRIPTION
**Pull request by leonmezu1**
Edited the readme file and the html basic document and  the header

### CODE REVIEW

**Comment by kitpao** [comment]
Can you review our project please? :)

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work team :clap:. However, some changes needs to be made in order for us to approve your project. check below :arrow_down: .

#### Page structure  

##### 3 articles rows

* [x] It contains 3 articles, each takes **one third of the width** (content is not important).

##### Footer

- I see that you didn't add a footer section :mag: , please Do.

#### General Requirements 

* [x] Check HTML markup with this validator https://validator.w3.org/. 
* [x] It contains centered links (content is not important).

### That's it :relieved: . When finished, kindly resubmit another code review request :pray:

**Comment by kitpao** [comment]
@stratospherique hello, thanks for your review, do you mean these articles?
image
or these ones?
image

**Comment by stratospherique** [comment]
- I would say yes since there is an ad there. Just try to add a row with 3 articles (each article takes one third of the rows width. :nerd_face:

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi Team, 👋👋
Your project looks nice, but it seems the same issues raised by the previous TSE still persist.🙂 
* There ought to be a row with an ```<article>``` where each article is one-third of the width🙂 
just like below
Screenshot from 2019-09-24 16-10-49

* You have to add a footer to the project so you can meet the requirement🙂 
just like below
Screenshot from 2019-09-24 16-11-11
## Do these changes and submit code for review.🙂

**Comment by kitpao** [comment]
@Forison oh, ok, sorry,
* we thought that by creating a grid with the articles in 1/3, it would be done. So we take out the advertisement space right?
image

* and we thought that only the important links were required in our modified footer section, so we have to put all the elements right?

**Comment by kitpao** [comment]
How about now?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+                <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li>
+              </ul>
+            </div>
+            <div class="final-img important">
+            </div>
+            <footer class="footer-links important">
+              <ul>
+                <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li>
+              </ul>
+            </footer>
```
Comment (Redvanisation) [inline index.html]: The `<footer>` tag shouldn't be inside the `<main>` tag each one of them represents a different part of the website content.

**Review by Redvanisation — APPROVED** [review]
### Hello @kitpao and @leonmezu1 🌟 

### _Nice work on this project, it seems that you guys have met the requirements so I am approving your project. 👍_ 

- However, there is a suggestion which I have made in **the comment below** that you should take into consideration in your future projects.

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

File: index.html (hunk 1 above)
Comment (kitpao) [reply index.html]: :O thank you!

Comment (leonmezu1) [reply index.html]: Thank you !

Comment (leonmezu1) [reply index.html]: Now the footer is out from the main section of the document. ✔

**Pull request merged**